CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ this release. The Web3Signer docker image will also use Java 21, however, binary
 require Java 21 to be available on the host machine.
 - This is the last Web3Signer release to use the "filecoin" mode. The "filecoin" mode will be removed in a future release.
 
+### Breaking Changes
+- Use Java 21 for build and runtime. Remove Java 17 variant of docker image. zip/tar.gz distributions will require Java 21 or above to run Web3Signer.
+
```

Comment:
Also, we need to mention the changing of metrics to use openmetrics standard, users will need to update their dashboard

Reply:
Updated changelog, let me know if wording needs to be changed.